lib/pure/lenientops.nim

```diff
@@ -25,32 +25,44 @@
 ## should make an explicit choice.
 
 proc `+`*[I: SomeInteger, F: SomeFloat](i: I, f: F): F {.noSideEffect, inline.} =
+  runnableExamples: static: doAssert 42 + 3.14 is SomeFloat
```

Comment:
Honestly I don't think lenientops needs documentation for individual procedures like this at all

Reply:
@juancarlospaco how about adding `runnableExamples` just for 1 proc in lenientops.